submissions/SerhiiYakovenko/html-css-popup/img/logo.svg

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 19.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
It is also good to use minifier for svg files, which will remove unnecessary things and make the file size smaller. For instance: https://jakearchibald.github.io/svgomg/